Makefile

```diff
@@ -38,16 +38,6 @@ JSL_FILES_NODE   = $(JS_FILES)
 JSSTYLE_FILES	 = $(JS_FILES)
 JSSTYLE_FLAGS    = -f tools/jsstyle.conf
 
-MAN_SECTION	= 1
-MAN_INROOT	= ./docs/man
-MAN_OUTROOT	= ./man
-include ./tools/mk/Makefile.manpages.defs
-
-MAN_SECTION	= 3
-MAN_INROOT	= ./docs/man
-MAN_OUTROOT	= ./man
-include ./tools/mk/Makefile.manpages.defs
-
 include ./tools/mk/Makefile.defs
 
 #
```

Comment:
Unrelated to your change (IOW, fine to leave this for anotehr change if you prefer), but while here: use `npm install` rather than `npm rebuild`. Lamely, I'm not entirely clear on the difference but in my experience `npm rebuild` is not exactly what is wanted and almost everything uses `npm install`.

Reply:
Actually, look at this:

```
[12:29:21 trentm@bluesteel:~/joy/node-buckets-mdapi (MANTA-4628)]
$ make
npm rebuild
[12:29:23 trentm@bluesteel:~/joy/node-buckets-mdapi (MANTA-4628)]
$ ls
CHANGES.md   Jenkinsfile  LICENSE      Makefile     README.md    deps         lib          package.json tools
```

`npm rebuild` doesn't do the "install" work at all. Definitely change this.

Reply:
Done